### ISSUE TITLE
Unrecognize valid urls in worker options

### ISSUE THREAD

**Issue by lqt93** [naptha/tesseract.js#558]
**Describe the bug**
When I assigned these below valid urls to paths in worker options, tesseract.js did not recognize them:
- http://someserver/abc (without any dot)
- http://somerserver:port/abc

**To Reproduce**
1. Assign url
``` typescript
import { createWorker } from "tesseract.js";
const url = "http://someserver/abc";
createWorker({
  langPath: url
})
```
2. Throw error

**Expected behavior**
Works like when assigning other valid urls.

**Screenshots**
![Screen Shot 2021-08-02 at 11 34 07](https://user-images.githubusercontent.com/8168339/127804785-937d5356-0895-4461-a338-8125588cb7c0.png)

**Desktop (please complete the following information):**
 - Browser: Chrome

**Additional context**
- I believe the problem is caused due to "is-url" library does not regcognize these urls as valid.

**Comment by Balearica** [comment]
Thanks for raising.  I see that you've also opened a Git Issue for the `is-url` library.  As I don't think doing our own URL validation is within the scope of this project, I don't think there's anything actionable here until (1) the `is_url` package is updated or (2) somebody can suggest a more reliable package we can use instead.

**Comment by Balearica** [comment]
I recently encountered this issue myself, and now believe it can be solved within Tesseract.js by cutting `is-url` from the browser version entirely.

The `is-url` dependency only makes sense in the context of Node.js.  For Node, a user may specify a string that is either (1) a URL or (2) a local path.  As these are not separate arguments, Tesseract.js needs to figure out which one it was provided, which it uses the `is-url` package to do.  However, in a browser we do not have local file access--any path provided by a user must be a URL.

**Comment by Balearica** [comment]
Closing as this should be resolved in the [v4.1.0](https://github.com/naptha/tesseract.js/releases/tag/v4.1.0) release.
